src/types/redis_hash.cc

```diff
@@ -418,4 +472,208 @@ rocksdb::Status Hash::RandField(const Slice &user_key, int64_t command_count, st
   return rocksdb::Status::OK();
 }
 
+rocksdb::Status Hash::ExpireFields(const Slice &user_key, uint64_t expire_ms, const std::vector<Slice> &fields,
```

Comment:
can remove `is_persist` by checking if `expire_ms` is equal to 0?

Reply:
Yes. i will change it

Reply:
I tried the behavior on redis. Maybe we need to distinguish the `HEXPIREAT` with 0 and `HPERSIST`.

For example:
- ` HEXPIREAT k 0 FIELDS 1 f ` will return `2` (equivalent expire a pass time)
- `HPERSIS k FIELDS 1 f` will return `1` (expiration was removed)

So, we need a flag to distinguish them?

Reply:
I think we should distinguish this in Commands instead of DB implementation.

Reply:
Yes. In redis source code. The hpersist command is a independent function. not reuse `hexpireGenericCommand`.

https://github.com/redis/redis/blob/unstable/src/t_hash.c#L3235-L3236